test/files/run/t9403/Test_2.scala

```diff
@@ -0,0 +1,37 @@
+import p.C
+import scala.tools.asm.Opcodes
+import scala.tools.partest.BytecodeTest
+import scala.tools.partest.ASMConverters._
+
+
+object Test extends BytecodeTest {
+  def foo(c: C, x: Int) = c.f(x)
+  def bar(c: C, x: Int) = c.g(x)
```

Comment:
`bar` should really be named `goo` for consistency. Really trying to find some way to be helpful here.